### PR TITLE
Add Bahasa Melayu Localizations

### PR DESCRIPTION
**Pull request by faizmokhtar**
- Referring to the issue #279, This PR added Bahasa Melayu (ms) localizations.

### CODE REVIEW

File: Quran/Base.lproj/Android.strings [hunk 1]
```diff
@@ -287,4 +287,4 @@ Please note that we receive lots of emails, so we may not be able to reply to al
 "repeatValues1" = "1 time";
 "repeatValues2" = "2 times";
 "repeatValues3" = "3 times";
-"repeatValues4" = "loop";
```
Comment (mohamede1945) [inline Quran/Base.lproj/Android.strings]: Please revert this change.

File: Quran/Base.lproj/Readers.strings [hunk 2]
```diff
@@ -1,15 +1,15 @@
-"qari_minshawi_murattal_gapless" = "Minshawi Murattal (gapless)";
```
Comment (mohamede1945) [inline Quran/Base.lproj/Readers.strings]: Please revert changes in this file.

File: Quran/Base.lproj/Suras.strings [hunk 3]
```diff
@@ -1,6 +1,6 @@
 "sura_names[0]" = "Al-Fatihah";
```
Comment (mohamede1945) [inline Quran/Base.lproj/Suras.strings]: Please revert changes in this file.

File: Quran/ms.lproj/Android.stringsdict [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            <key>NSStringLocalizedFormatKey</key>
+            <string>%#@files_downloaded@</string>
+            <key>files_downloaded</key>
+            <dict>
+                <key>NSStringFormatSpecTypeKey</key>
+                <string>NSStringPluralRuleType</string>
+                <key>NSStringFormatValueTypeKey</key>
+                <string>d</string>
+                <key>one</key>
+                <string>1 sura downloaded</string>
```
Comment (mohamede1945) [inline Quran/ms.lproj/Android.stringsdict]: This is not translated. Other values in this file also are not translated.

File: Quran/ms.lproj/Readers.strings [hunk 5]
```diff
@@ -0,0 +1,50 @@
+"qari_minshawi_murattal_gapless" = "Minshawi Murattal (gapless)";
```
Comment (mohamede1945) [inline Quran/ms.lproj/Readers.strings]: Should you translate `gapless`?

File: Quran/ms.lproj/Suras.strings [hunk 6]
```diff
@@ -0,0 +1,114 @@
+"sura_names[0]" = "Al-Fatihah";
```
Comment (mohamede1945) [inline Quran/ms.lproj/Suras.strings]: If there are no changes than the base file, then let's not have this file.

File: Quran/Base.lproj/Android.strings (hunk 1 above)
Comment (faizmokhtar) [reply Quran/Base.lproj/Android.strings]: Reverted

File: Quran/Base.lproj/Readers.strings (hunk 2 above)
Comment (faizmokhtar) [reply Quran/Base.lproj/Readers.strings]: Reverted.

File: Quran/Base.lproj/Suras.strings (hunk 3 above)
Comment (faizmokhtar) [reply Quran/Base.lproj/Suras.strings]: Reverted.

File: Quran/ms.lproj/Android.stringsdict (hunk 4 above)
Comment (faizmokhtar) [reply Quran/ms.lproj/Android.stringsdict]: I have updated the translations for this file.

File: Quran/ms.lproj/Readers.strings (hunk 5 above)
Comment (faizmokhtar) [reply Quran/ms.lproj/Readers.strings]: I have updated the translations for `(gapless)`

File: Quran/ms.lproj/Suras.strings (hunk 6 above)
Comment (faizmokhtar) [reply Quran/ms.lproj/Suras.strings]: I have updated the name of the surahs based on its Malay localizations.

**Comment by mohamede1945** [comment]
Jazak Allah khyran for updating it and so sorry for late reply.
